.gitignore

```diff
@@ -83,4 +83,4 @@ obj/
 bin/
 
 # Ignore user uploads
-uploads/
\ No newline at end of file
+uploads/
```

Comment:
You did add this, but you are still tracking files in this folder. See this PR's total changes:
![image](https://github.com/svsticky/plakplaats/assets/69711739/f723f593-4ad7-4b5b-9b52-ee7b26386abe)

You should briefly unignore these files, delete them, commit those changes, and start ignoring the folder again.